fiftyone/server/routes/sample.py

```
@@ -645,10 +645,113 @@ async def patch(self, request: Request, data: List[dict]) -> JSONResponse:
+                detail="Detection has no in-database mask to commit",
+            )
+
+        mask_path = detection.mask_path
+
+        try:
+            detection.export_mask(
+                mask_path, update=True, overwrite_path=True
+            )
+            save_sample(sample, if_last_modified_at)
+        except Exception as err:
+            logger.error("Failed to commit mask to %s: %s", mask_path, err)
```

Ruff (0.15.11)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)